### PR TITLE
Merge preview 7 to main

### PR DESCRIPTION
**Pull request by sfoslund**

### CODE REVIEW

File: src/Cli/dotnet/commands/dotnet-build/BuildCommandParser.cs [hunk 1]
```diff
@@ -49,6 +55,10 @@ public static Command GetCommand()
             command.AddOption(NoIncrementalOption);
             command.AddOption(NoDependenciesOption);
             command.AddOption(NoLogoOption);
+            command.AddOption(SelfContainedOption);
+            command.AddOption(NoSelfContainedOption);
+			command.AddOption(CommonOptions.ArchitectureOption());
```
Comment (pranavkm) [inline src/Cli/dotnet/commands/dotnet-build/BuildCommandParser.cs:60]:
```suggestion
			   command.AddOption(CommonOptions.ArchitectureOption());
```

**Comment by pranavkm** [comment]
@javiercn could you verify if your SWA changes look good?

File: src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets [hunk 2]
```diff
@@ -1073,6 +1079,23 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
+  <!--
+  ============================================================
+  Command Line Confirugation Validation
```
Comment (dsplaisted) [inline src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets:1084]: Typo which I missed in code review:
```suggestion
  Command Line Configuration Validation
```

**Review by dsplaisted — APPROVED** [review]

**Comment by sfoslund** [comment]
@javiercn can you please review so we can get this merged?

**Pull request merged**